config.template/faf-java-api/faf-java-api.env

```diff
@@ -64,3 +64,6 @@ STEAM_LINK_ERROR_URL=http://localhost:8020/error?code=%s
 STEAM_LINK_REDIRECT_URL=http://localhost:8010/users/linkToSteam?token=%s
 STEAM_LINK_SUCCESS_URL=http://localhost:8020/linked_to_steam
 TUTORIAL_THUMBNAIL_URL_FORMAT=https://localhost:8091/faf/tutorials/thumbs/%s
+TESTING_EXE_UPLOAD_KEY=banana
```

Comment:
Maybe leave it empty so the endpoint does not get activated by default and make sure java-api deactivates endpoint if this is empty. I am certain one day someone forgets this feature and it will be set live in producton with passowrd banana which is pretty bad....

Reply:
Well, if we have banana below, i think, we should have it here too.